docs/reference/configuration.md

```diff
@@ -149,112 +262,73 @@ gateways:
 See [Airflow Integration Guide](../integrations/airflow.md) for information about how to integrate Airflow with SQLMesh.
 
 #### Cloud Composer
+
+The Google Cloud Composer scheduler type shares the same configuration options as the `airflow` type, except for `username` and `password`. Cloud Composer relies on `gcloud` authentication, so the `username` and `password` options are not required.
+
+Example configuration:
+
 ```yaml linenums="1"
-scheduler:
-    type: cloud_composer
-    airflow_url: <airflow_url>
+gateways:
+    my_gateway:
+        scheduler:
+            type: cloud_composer
+            airflow_url: <airflow_url>
 ```
-This scheduler type shares the same configuration options as the `airflow` type, except for `username` and `password`.
-Cloud Composer relies on `gcloud` authentication, so the `username` and `password` options are not required.
 
-See [Airflow Integration Guide](../integrations/airflow.md) for information on how to integrate Airflow with SQLMesh.
+## Gateway/connection defaults
+
+The default gateway and connection keys specify what gateway/connection should be used if a command does not explicitly specify one or some gateways in the `gateways` dictionaries do not specify every connection.
```

Comment:
> if a command does not explicitly

it's not exactly clear what's "command" here

Reply:
I'm finding this language tricky - see what you think about the updated version.